cmd/icingadb/main.go

```diff
@@ -317,6 +318,14 @@ func run() int {
 
 // checkDbSchema asserts the database schema of the expected version being present.
 func checkDbSchema(ctx context.Context, db *icingadb.DB) error {
+	var expectedDbSchemaVersion uint16
+	switch db.DriverName() {
+	case "icingadb-mysql":
```

Comment:
I would introduce driver constants for all `icingadb-*` occurrences as in 71f3b494d15ce4ada2e330cdc70fc624923db223.

Reply:
Let's see if/when that gets merged... wait! History **retention**? Never mind, other construction area.

Reply:
This has nothing to do with history retention. It introduces constants for the driver names we support that actually serve a purpose, rather than just using a random looking string over and over again. Also, constants make it easier to see the usage and what's actually available.

Reply:
OK. Btw. PGSQL shouldn’t exist in 71f3b494d15ce4ada2e330cdc70fc624923db223.